.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8"
```

Comment:
an engine/platform change should warrant a major update.  any reason why we're dropping 8?

Reply:
I'm testing, we won't be dropping node 8. This PR is still work in progress thus the WIP in the title.

Reply:
We can still add support to 11 if that works out of the box.

Reply:
Sure, let's do it in another PR.